website/package.json

```diff
@@ -27,6 +28,7 @@
     "@hashicorp/react-text-split-with-image": "^4.2.5",
     "@hashicorp/react-vertical-text-block-list": "^7.0.0",
     "adm-zip": "^0.5.5",
+    "fathom-client": "^3.2.0",
```

Comment:
Can this dep be cut?

Reply:
🤔 I think it can, actually!  I had been mistaken thinking this should be an explicit dep. The package installed as a dep of `@hashicorp/platform-analytics` should suffice.